### PR TITLE
Add doc about frameworks and libraries

### PR DESCRIPTION
**Pull request by rabbit008**
Add doc about frameworks and libraries

### CODE REVIEW

File: docs/en_US/SupportedFramework_Library.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Framework and Library Supported 
+NNI provides Python API, supporting all framework models and libraries written in python (`version >= 3.5`), and we have implemented many related examples and detailed their principles and running steps.
```
Comment (xuehui1991) [inline docs/en_US/SupportedFramework_Library.md]: supporting all framework models -> supporting all frameworks?

File: docs/en_US/SupportedFramework_Library.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+
+## Support Framework:
```
Comment (xuehui1991) [inline docs/en_US/SupportedFramework_Library.md]: For examples?

File: docs/en_US/SupportedFramework_Library.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+* [CNTK (Python language)]https://github.com/microsoft/CNTK
+* [Scikit-learn]https://scikit-learn.org/stable/
+    <ul>
+    <li><a href="TrialExample/SklearnExamples.md">Scikit-learn</a><br/></li>
+    </ul>
+* [Spark MLlib]http://spark.apache.org/mllib/
+* [Chainer]https://chainer.org/
+* [Theano]https://pypi.org/project/Theano/
+
+## Support Library:
```
Comment (xuehui1991) [inline docs/en_US/SupportedFramework_Library.md]: no need to separate here. And seems 'Sklearn' is a library, too.

File: docs/en_US/SupportedFramework_Library.md (hunk 1 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: fixed.

File: docs/en_US/SupportedFramework_Library.md (hunk 3 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: fixed,I have moved sklearn to the library.

File: docs/en_US/SupportedFramework_Library.md (hunk 2 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: You mean by merging the framework with the library, and then writing a general title:For Example?

File: docs/en_US/SupportedFramework_Library.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Framework and Library Supported
```
Comment (scarlett2018) [inline docs/en_US/SupportedFramework_Library.md]: supports

File: docs/en_US/SupportedFramework_Library.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Framework and Library Supported 
+NNI provides Python API, supporting all frameworks and libraries written in python (`version >= 3.5`), and we have implemented many related examples and detailed their principles and running steps.
```
Comment (scarlett2018) [inline docs/en_US/SupportedFramework_Library.md]: With the built-in Python API, NNI naturally supports the hyper parameter tuning and neural network search for all the AI frameworks and libraries who support Python models. NNI had also provided a set of examples and tutorials for some of the popular scenarios to make jump start easier.

File: docs/en_US/SupportedFramework_Library.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+
+
+## Support Framework:
+
+NNI supports all frameworks, as long as they are written in python. Some common frameworks are listed below, including NNI example tutorials based on them.
```
Comment (scarlett2018) [inline docs/en_US/SupportedFramework_Library.md]: remove this sentence, seems dup with the above one.

File: docs/en_US/SupportedFramework_Library.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (scarlett2018) [inline docs/en_US/SupportedFramework_Library.md]: **Supported AI Frameworks:**

File: docs/en_US/SupportedFramework_Library.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+      <li><a href="../../examples/trials/network_morphism/README.md">Network_morphism</a><br/></li>
+    </ul>
+* [MXNet]https://github.com/apache/incubator-mxnet
+* [Caffe2]https://github.com/BVLC/caffe
+* [CNTK (Python language)]https://github.com/microsoft/CNTK
+* [Spark MLlib]http://spark.apache.org/mllib/
+* [Chainer]https://chainer.org/
+* [Theano]https://pypi.org/project/Theano/ <br/>
+
+In addition to these, other frameworks written in python are also supported by NNI.
```
Comment (scarlett2018) [inline docs/en_US/SupportedFramework_Library.md]: You are encouraged to contribute more examples for other NNI users. //add the how-to contribute link.

File: docs/en_US/SupportedFramework_Library.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+* [CatBoost]https://catboost.ai/
+* [LightGBM]https://lightgbm.readthedocs.io/en/latest/
+    <ul>
+    <li><a href="TrialExample/GbdtExample.md">Auto-gbdt</a><br/></li>
+    </ul>
+Here is just a small list of libraries that supported by NNI. If you are interested in NNI, you can refer to the [tutorial](TrialExample/Trials.md) to complete your own hacks.
+
+
+
+In addition to these experiments, we also welcome more and more users to apply NNI to your own experiment, if you have any doubts, please refer [Write a Trial Run on NNI](TrialExample/Trials.md). In particular, if you want to be a contributor of NNI, whether it is the sharing of examples , writing of Tuner or otherwise, we are all looking forward to your participation.More information please refer to [here](Tutorial/Contributing.md).
```
Comment (scarlett2018) [inline docs/en_US/SupportedFramework_Library.md]: this .md is for Training service not for experiments, please tune the wordings.

File: docs/en_US/SupportedFramework_Library.md (hunk 4 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: Fixed

File: docs/en_US/SupportedFramework_Library.md (hunk 5 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: Fixed

File: docs/en_US/SupportedFramework_Library.md (hunk 8 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: fixed

File: docs/en_US/SupportedFramework_Library.md (hunk 7 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: fixed

File: docs/en_US/SupportedFramework_Library.md (hunk 6 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: fixed

File: docs/en_US/SupportedFramework_Library.md (hunk 9 above)
Comment (rabbit008) [reply docs/en_US/SupportedFramework_Library.md]: fixed

**Pull request merged**